Sources/PointFree/Session.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+import HttpPipeline
+import Prelude
+import Tuple
+
+public struct Session: Codable {
+  public var flash: Flash?
+  public var userId: Database.User.Id?
+
+  public static let empty = Session(flash: nil, userId: nil)
+
+  private enum CodingKeys: String, CodingKey {
+    case userId = "user_id"
```

Comment:
we could also let this encode as `userId` right? any reason to do snake case?

Reply:
just convention, lol. do we want all our internal stuff camel? including postgres?

Reply:
postgres seems to prefer snack case, but in json world I think we can decide. im fine with either, was just realizing that we do this snake case stuff when it isnt technically required.